src/main/java/zeus/command/TaskDoneCommand.java

```diff
@@ -0,0 +1,47 @@
+package zeus.command;
+
+import zeus.exception.ZeusException;
+import zeus.main.Storage;
+import zeus.main.TaskList;
+import zeus.main.Ui;
+
+/**
+ * Class that handles the case of marking a Task as done.
+ */
+
+public class TaskDoneCommand extends Command {
+
+    private int idx;
+
+    /**
+     * Constructor for TaskDoneCommand.
+     *
+     * @param idx
```

Comment:
```suggestion
     * @param idx The index of the task to be marked as done.
```

Should a javadocs description be included for the parameter?